bin/plot_tree.py

```diff
@@ -54,7 +54,8 @@ def annotation_timepoints(annotation):
         return set(filter(lambda x: x, annotation.get('cluster_timepoints',
             annotation['timepoints']).split(':')))
     timepoints = sorted(reduce(set.union, (annotation_timepoints(x) for x in annotations.values()), set()))
-    colors = ['#%02x%02x%02x' % c for c in colorbrewer.RdYlBu[max(len(timepoints), 3)]]
+    palette = colorbrewer.RdYlBu[11] # this is the max # of colors in this set
```

Comment:
@eharkins I don't think we want to do it this way. Always using 11 here means that when we only have two or three timepoints, the colors chosen will end up being much closer together in color space than they could be if we dynamically base on the number of timepoints, as above.

Reply:
Ah I didnt think about that it is a continuous color scale. That makes sense. That being said, why is it a continuous color scale? I previously thought of timepoints as categorical; are you trying to convey timepoints on a timeline (eg. where red=old and blue=recent)?

Reply:
Ideally things would match up by order so that the continuity of it makes sense. But even for categorical color scales, you're more likely to get a "well spread" set of colors if you pick the right colorbrewer color set for the number of categories you actually have.

There's a whole other question around how you might more properly associate order with timepoint ids, but I don't know that it's worth worrying about it too much in CFT. Olmsted however could benefit from this.